src/core/frontend/applets/mii.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <functional>
+
+#include "core/hle/result.h"
+#include "core/hle/service/mii/mii_manager.h"
+
+namespace Core::Frontend {
+
+struct MiiParameters {
+    bool editable;
+    Service::Mii::MiiInfo mii_data{};
+};
+
+class MiiApplet {
+public:
+    virtual ~MiiApplet();
+
+    virtual void ShowMii(
+        const MiiParameters& parameters,
+        std::function<void(Core::Frontend::MiiParameters parameters)> callback) const = 0;
```

Comment:
```suggestion
        const std::function<void(const Core::Frontend::MiiParameters& parameters)>& callback) const = 0;
```

Otherwise this copies 88 bytes every time the function is called (on top of copying the function instance itself)